Makefile

```diff
@@ -1,10 +1,18 @@
 DOCKER_IMAGE ?= lev-report
+test_image = quay.io/ukhomeofficedigital/lev-report-test:latest
```

Comment:
I don't see why we want a separate test image. Surely we can just run `npm run test:acceptance` (or whatever we choose to call it)?

Reply:
Ah, maybe because we need to do so inside the docker-compose network? Even so I'd think we should be able to run a local container rather than pushing anything specific to Quay.io.

Reply:
Then again we might need something at some point for e2e testing. But I think those tests will be simpler than the accptance tests. (Also we always want to run the latest acceptance tests rather than pulling some older version from Quay.io.)

Reply:
Removed for now - will tackle in a separate edit later...